doc/concepts.rst

```diff
@@ -696,6 +701,15 @@ of your project. What you should write instead is:
      (deps   blah.mll)
      (action (chdir %{workspace_root} (run ocamllex -o %{target} %{deps}))))
 
+.. _dune-action-plugin:
+
+Dune action plugin
+==================
+
+``Dune-action-plugin`` provides a monadic interface to express program
+dependencies directly inside the source code. Programs using this feature
+should be declared using ``dynamic-run`` construction instead of usual ``run``.
+
```

Comment:
This section should be moved to `dune-libs.rst`.

Reply:
Moved.